addons/auth_signup/data/auth_signup_data.xml

```diff
@@ -195,6 +195,98 @@
             <field name="user_signature" eval="False"/>
         </record>
 
+        <!-- Email template for reminder of unregistered users -->
+        <record id="unregistered_user_reminder" model="mail.template">
+            <field name="name">Auth Signup: Unregistered Users</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">Reminder for unregistered users</field>
+            <field name="email_from">"${object.company_id.name|safe}" &lt;${object.company_id.email or user.email|safe}&gt;</field>
+            <field name="email_to">${object.email|safe}</field>
+            <field name="body_html" type="html">
```

Comment:
Instead of hardcoding the template layout, use notif_layout parameter from template send_mail. That way we don't have synchronization issues or templates having "more or less" the same code.

Reply:
@tde-banana-odoo , contents are different in header than our main notif layout(mail_notification_light) so we hardcoded template. if we avoid some header stuff then we can use the notif_layout. Thanks

Reply:
What are the differences exactly ?

Reply:
@tde-banana-odoo , in main view header, will be like
`your XYZ` and subtitle will be `reocrd_name` while in spec  header: 
header `Name of your database`  and Big `Pending invitations`
Thanks